src/lib/kclHelpers.ts

```diff
@@ -0,0 +1,97 @@
+import { err } from './trap'
+import { engineCommandManager } from 'lib/singletons'
+import { parse, ProgramMemory, resultIsOk } from 'lang/wasm'
+import { PrevVariable } from 'lang/queryAst'
+import { executeAst } from 'lang/langHelpers'
+import { KclExpression } from './commandTypes'
+
+const DUMMY_VARIABLE_NAME = '__result__'
+
+export function programMemoryFromVariables(
+  variables: PrevVariable<string | number>[]
+): ProgramMemory | Error {
+  const memory: ProgramMemory = ProgramMemory.empty()
```

Comment:
You probably want `programMemoryInit()` instead so that it includes the prelude. `HALF_TURN`, etc.

Reply:
Hell yeah thanks